### DISCUSSION TITLE
[IntelliSense] autocomplete and preview for inline ruby (erb)

### DISCUSSION THREAD

**Discussion by ShadowCrafter011** [tailwindlabs/tailwindcss#12838] [Ideas]
I really like the autocomplete and css preview on html classes. But I am using Ruby on Rails and hence my html files are erb. The autocomplete and preview work very well on html classes but if I create html elements with embeded ruby it does not work.

Here is an example
```
<div class="w-full">
    <%= content_tag :div, "Hello World!", class: "bg-white" %>
</div>
```
The preview and autocomplete work on `w-full` but not on `bg-white`.

It would be nice if that feature could be added for `.html.erb` files.

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Comment by aviflombaum** [comment]
This worked for me:

```json
  "tailwindCSS.experimental.classRegex": ["class:\\s*\"([^\"]*)\"", "class:\\s*'([^']*)'"],
```
